### DISCUSSION TITLE
Are Skywalking 9.1.0 and Elasticsearch 8.11.3 compatible?

### DISCUSSION THREAD

**Discussion by xiaoqiaoqiao888** [apache/skywalking#12281] [Q&A] [answered]
Are Skywalking 9.1.0 and Elasticsearch 8.11.3 compatible?
skywalking 9.1.0 和elasticsearch 8.11.3版本适配么？

**Comment by wu-sheng** [comment, marked as answer]
Duplicated with https://github.com/apache/skywalking/issues/12224
